apps/src/feedback.js

```diff
@@ -1466,9 +1466,14 @@ FeedbackUtils.prototype.hasAllBlocks_ = function (blocks) {
  * @return {Array<Object>} The blocks.
  */
 FeedbackUtils.prototype.getUserBlocks_ = function () {
-  var allBlocks = Blockly.mainBlockSpace.getAllUsedBlocks();
-  var blocks = allBlocks.filter(function (block) {
-    var blockValid = !block.disabled && block.type !== 'when_run';
+  const allBlocks = [
```

Comment:
nit: since we reuse this logic should we make a helper function that gets all blocks for the workspaces?

Reply:
Will do as a follow-up!